helm-charts/metal3-deploy/Chart.lock

```diff
@@ -0,0 +1,18 @@
+dependencies:
```

Comment:
Does this file need to be included in the repo?

Reply:
It was also being ignored by the .gitignore file...

Reply:
Good question. I think we will have to check in this file because this is the snapshot of the dependencies.